src/test/features/claim/routes/check-and-send.ts

```diff
@@ -136,6 +356,186 @@ describe('Claim issue: check and send page', () => {
           .send({ signed: 'true' })
           .expect(res => expect(res).to.be.redirect.toLocation(ClaimPaths.startPaymentReceiver.uri))
       })
+
+      it('Should validate that a claim made by individual against soleTrader and redirect to payment page', async () => {
+        draftStoreServiceMock.resolveFind('claim',
+          { defendant: { ...draftStoreServiceMock.sampleClaimDraftObj.defendant, partyDetails: soleTraderDetails } })
+
+        await request(app)
+          .post(ClaimPaths.checkAndSendPage.uri)
+          .send({ type: SignatureType.BASIC })
+          .set('Cookie', `${cookieName}=ABC`)
+          .send({ signed: 'true' })
+          .expect(res => expect(res).to.be.redirect.toLocation(ClaimPaths.startPaymentReceiver.uri))
+      })
+
```

Comment:
We only need 1 test for post redirecting to correct location. I am sorry but I can't see any benefit in below tests.

Reply:
Agreed. the posting of this page just verifies the statement of truth. it does not do any checks on the draft data.

Reply:
removed